src/main/java/com/example/TechItEasy/controllers/ExceptionsController.java

```diff
@@ -0,0 +1,27 @@
+package com.example.TechItEasy.controllers;
+
+import com.example.TechItEasy.exceptions.RecordNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class ExceptionsController {
+    @ExceptionHandler(value = RecordNotFoundException.class)
+    public ResponseEntity<Object>
```

Comment:
De response body (de eerste parameter van de ResponseEntity constructor) is hier `exception.getMEssage()`, dat is een String. De returntype van deze methode, mag je daarom ook `ResponseEntity<String>` maken ipv `ResponseEntity<Object>`. Object werkt ook, maar String is preciezer en daarom beter.